Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -0,0 +1,2679 @@
+            'InstanceId': instance['InstanceId'],
+            'MonitoringState': instance['Monitoring']['State']
+        })
+
+    ec = {'AWS.EC2.Instances(val.InstancesId === obj.InstancesId)': response['InstanceMonitorings']}
+    human_readable = tableToMarkdown('AWS EC2 Instances', data)
+    return_outputs(human_readable, ec)
+
+
+def unmonitor_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    data = []
+    response = client.unmonitor_instances(InstanceIds=parse_resource_ids(args.get('instancesIds')))
+
+    for instance in response['InstanceMonitorings']:
+        data.append({
+            'InstanceId': instance['InstanceId'],
+            'MonitoringState': instance['Monitoring']['State']
+        })
+
+    ec = {'AWS.EC2.Instances(val.InstancesId === obj.InstancesId)': response['InstanceMonitorings']}
+    human_readable = tableToMarkdown('AWS EC2 Instances', data)
+    return_outputs(human_readable, ec)
+
+
+def reboot_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.reboot_instances(InstanceIds=parse_resource_ids(args.get('instanceIds')))
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Instances were rebooted")
+
+
+def get_password_data_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.get_password_data(InstanceId=args.get('instanceId'))
+    data = {
+        'InstanceId': response['InstanceId'],
+        'PasswordData': response['PasswordData'],
+        'Timestamp': datetime.strftime(response['Timestamp'], '%Y-%m-%dT%H:%M:%SZ')
+    }
+
+    ec = {'AWS.EC2.Instances(val.InstancesId === obj.InstancesId).PasswordData': data}
+    human_readable = tableToMarkdown('AWS EC2 Instances', data)
+    return_outputs(human_readable, ec)
+
+
+def modify_network_interface_attribute_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    kwargs = {'NetworkInterfaceId': args.get('networkInterfaceId')}
+
+    if args.get('sourceDestCheck') is not None:
+        kwargs.update({'SourceDestCheck': {'Value': True if args.get('sourceDestCheck') == 'True' else False}})
+    if args.get('attachmentId') is not None and args.get('deleteOnTermination') is not None:
+        kwar
```

Comment:
this can throw value error. better calculate it before the append and in a try block.